paf-mvp-frontend/tests/test-cases/onekey-lib.test.ts

```diff
@@ -21,8 +25,21 @@ const pafClientNodeHost = 'http://localhost';
 let lib: OneKeyLib;
 let notificationHandler: jest.Mock<Promise<void>, []>;
 
+const auditLogStorageService = {
```

Comment:
Maybe setting the type would help to update the test if there is a change in the AuditLogStorageService thanks to an alert of the transpiler.

`````
const auditRepo: AuditRepository = {
  // ...
};
`````

Reply:
Totally agree.